### ISSUE TITLE
Low validation accuracy of the converted pytorch model from tensorflow

### ISSUE THREAD

**Issue by lith0613** [Microsoft/MMdnn#520]
I have converted the tensorflow pre-trained model from pytorch as the instruction and loaded the model in my pytorch demo script.  I used the converted resnet_v1_50 on the ImageNet validation set, and I found that the top1 accuracy was just 0.62%.  However, the top1 accuracy of the original pytorch pretrained resnet_50 is 76.15%, why I get only 0.62% top1 accuray based on the converted model ?

The main part of demo code is as follows: 

trainloader = imagenet_traindata(args.batch_size)
testloader = imagenet_testdata(args.batch_size)
 
MainModel = imp.load_source('MainModel', "tf_resnetv1_50_to_pth.py")
model = torch.load('tf_resnetv1_50_to_pth.pth')
 
model = nn.DataParallel(model)
model = model.cuda()
print(model)
 
trainloader = imagenet_traindata(args.batch_size)
testloader = imagenet_testdata(args.batch_size)
 
MainModel = imp.load_source('MainModel', "tf_resnetv1_50_to_pth.py")
model = torch.load('tf_resnetv1_50_to_pth.pth')
 
model = nn.DataParallel(model)
model = model.cuda()
print(model)
 
criterion = nn.CrossEntropyLoss()
optimizer = torch.optim.SGD(model.parameters(), lr=0.1, momentum=0.9, weight_decay=1e-4)
 
def validate(val_loader, model, criterion):
    losses = AverageMeter()
    top1 = AverageMeter()
    top5 = AverageMeter()
    model.eval()
 
    for i, (input, target) in enumerate(val_loader):
        input_var = torch.autograd.Variable(input, volatile=True).cuda()
        target_var = torch.autograd.Variable(target, volatile=True).cuda()
 
        # compute output
        output = model(input_var)
        loss = criterion(output, target_var)
 
        # measure accuracy and record loss
        prec1, prec5 = accuracy(output.data, target_var.data, topk=(1, 5))
        losses.update(loss.data[0], input.size(0))
        top1.update(prec1[0], input.size(0))
        top5.update(prec5[0], input.size(0))
 
    print('Test: *Loss {loss.avg:.4f} \tPrec@1 {top1.avg:.3f}'.format(loss=losses, top1=top1))
    return top1.avg
 
validate(testloader, model, criterion)
